settings.yaml

```diff
@@ -6,15 +6,16 @@ move_cost: 2
 batch_size: 32
 
 start_num_creatures: 6
+start_num_wolves: 2
 start_grass_fraction: 0.3
 start_rock_fraction: 0.05
 start_water_fraction: 0.01
 grass_grow_per_turn: 3
-world_size: 60
-scale: 10
+world_size: 35
+scale: 14
```

Comment:
Let's not merge these changes unless we have a good reason

Reply:
No reason indeed. I'll change them back.